labs/lab-2-javascript-game/script.js

```diff
@@ -0,0 +1,46 @@
+function getDamage(damage) {
+    return Math.floor(Math.random() * 10);
+
+}
+
+function startGame() {
+    const play = window.prompt("Do you want to play the game?");
+    if (play === "Yes") {
+        var name = window.prompt("Please enter your name:");
+        startCombat(name);
+    }
+
+    function startCombat() {
+        var winTotal = 0;
+        var grantHP = 10;
+        var userHP = 40;
+        var anotherAction;
+
+        while (winTotal < 5 && (userHP > 0)) {
```

Comment:
Have you set the winTotal to be < 5 here on purpose? The instructions ask for 3...